packages/mobile-sdk-alpha/android/src/main/java/com/selfxyz/selfSDK/ui/CameraFragment.kt

```diff
@@ -0,0 +1,417 @@
+package com.selfxyz.selfSDK.ui
```

Comment:
@seshanthS can we please add license headers to the kt files we create. realize that we don't have a lot so maybe we can just add manually instead of extending the node scripts.

```
// SPDX-FileCopyrightText: 2025 Social Connect Labs, Inc.
// SPDX-License-Identifier: BUSL-1.1
// NOTE: Converts to Apache-2.0 on 2029-06-11 per LICENSE.

```